app/views/pages/_card.html.erb

```diff
@@ -1,5 +1,5 @@
 <div id="<%= dom_id(card) %>"
-     class="card small-12 medium-<%= card.columns %> column margin-bottom end" data-equalizer-watch>
+     class="card small-12 medium-<%= card.columns %> column margin-bottom end">
```

Comment:
I see this partial and the one in `welcome/_card` are identical :thinking:. Would it make sense to use one file instead, under either `cards/_card` or `shared/_card`? That way we wouldn't have to change the code in two places :smile:.

Reply:
Yes, one file it's better! I've created two new partials `shared/_cards` and `shared/_card`. 😌